### PR TITLE
doc 88783 explain offline access req

### PR DESCRIPTION
**Pull request by mystiberry-okta**
@tthompson-okta  you commented on this work for OKTA-88783 once before, but then we split up oidc.md into two docs, and the conflict was so messy that I nuked that PR and redid these minor changes. Can you review?

Previous PR: https://github.com/okta/okta.github.io/pull/237

@royalchan-okta you should probably give the file a once-over to ensure I haven't made any git-related mistakes. Thanks!

### CODE REVIEW

File: _source/docs/api/resources/oauth2.md [hunk 1]
```diff
@@ -161,6 +161,11 @@ http://www.example.com/#error=invalid_scope&error_description=The+requested+scop
 
 The API takes an authorization code or a Refresh Token as the grant type and returns back an Access Token, ID Token and a Refresh Token.
 
+For web and native application types:
```
Comment (royalchan-okta) [inline _source/docs/api/resources/oauth2.md]: Missing the heading

`#####Refresh Tokens`

File: _source/docs/api/resources/oauth2.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ Use the postMessage() data model to help you when working with the <em>okta_post
 Parameter         | Description                                                                                        | DataType  | 
 ----------------- | -------------------------------------------------------------------------------------------------- | ----------| 
 id_token          | The ID Token JWT contains the details of the authentication event and the claims corresponding to the requested scopes. This is returned if the `response_type` includes `id_token`. | String   |
-access_token      | The `access_token` used to access the /userinfo endpoint. This is returned if the `response_type` included a token. <b>Important</b>: Unlike the ID Token JWT, the `access_toekn` structure is specific to Okta, and is subject to change. | String    |
+access_token      | The <em>access_token</em> used to access the [`/oauth2/v1//userinfo`](/docs/api/resources/oidc.html#get-user-information) endpoint. This is returned if the <em>response_type</em> included a token. <b>Important</b>: Unlike the ID Token JWT, the <em>access_token</em> structure is specific to Okta, and is subject to change. | String    |
```
Comment (royalchan-okta) [inline _source/docs/api/resources/oauth2.md]: double `/`

File: _source/docs/api/resources/oauth2.md [hunk 3]
```diff
@@ -91,8 +91,8 @@ Irrespective of the response type, the contents of the response is always one of
...
 ----------------- | -------------------------------------------------------------------------------------------------- | ----------| 
 id_token          | The ID Token JWT contains the details of the authentication event and the claims corresponding to the requested scopes. This is returned if the <em>response_type</em> includes <em>id_token</em> .| String    | 
-access_token      | The access_token that be used to access the userinfo endpoint. This is returned if the <em>response_type</em>  included token. Unlike the ID Token JWT, the access_token structure is Okta internal only and is subject to change.| String  |
-token_type        | This is always <em>Bearer</em> and is returned only when <em>token</em> is specified as a <em>response_type</em>. | String |
+access_token      | The <em>access_token</em> that is used to access the [`/oauth2/v1//userinfo`](/docs/api/resources/oidc.html#get-user-information) endpoint. This is returned if the <em>response_type</em>  included a token. Unlike the ID Token JWT, the <em>access_token</em> structure is specific to Okta, and is subject to change.| String  |
```
Comment (royalchan-okta) [inline _source/docs/api/resources/oauth2.md]: double `/`

File: _source/docs/api/resources/oauth2.md (hunk 1 above)
Comment (mystiberry-okta) [inline _source/docs/api/resources/oauth2.md]: So, Add the heading back (where, exactly?) and move the new stuff ("For web..." to the end of step 2") under it. ? @tthompson-okta okay?

Comment (tthompson-okta) [inline _source/docs/api/resources/oauth2.md]: Sounds right

**Comment by mystiberry-okta** [comment]
@royalchan-okta is this correct?

**Comment by mystiberry-okta** [comment]
@royalchan-okta can you let me know if I can ship it?

**Comment by royalchan-okta** [comment]
looks good, 🐑

**Pull request merged**